string obj_id  
 
---

SOMObservation object
